contrib/pai_vscode/README.md

```diff
@@ -22,18 +22,18 @@ OpenPAI VS Code Client is an extension to connect PAI clusters, submit AI jobs,
 
 Before using OpenPAI VS Code Client, follow below steps connecting to an OpenPAI cluster.
 
-Notice, it needs the version OpenPAI cluster >= 0.8.0
+Notice, the version of OpenPAI cluster must equal or greater than 0.8.0.
 
 1. Use shortcut key Ctrl+Shift+P to open command palette.
 2. Input and look for *PAI: Add PAI Cluster* as below.
 
     ![add cluster](https://raw.githubusercontent.com/Microsoft/pai/master/contrib/pai_vscode/assets/add_cluster.png)
 
-3. Press *Enter*, and it asks to input the host of OpenPAI. It can be domain name or IP Address. After that, press enter again.
+3. Press *Enter*, and input the host of an OpenPAI cluster. It can be domain name or IP Address. After that, press *Enter* again.
```

Comment:
```suggestion
3. Press <kbd>Enter</kbd>, and input the host of an OpenPAI cluster. It can be domain name or IP Address. After that, press <kbd>Enter</kbd> again.
```

Reply:
updated